build.gradle

```diff
@@ -99,6 +99,7 @@ dependencies {
     compile "com.ldtteam:datagenerators:0.1.38-ALPHA"
     
     implementation fg.deobf("mezz.jei:jei-1.16.4:${config.jei_version}")
+    compileOnly fg.deobf("curse.maven:resourcefulbees-384508:3206649")
```

Comment:
No.

Dependencies from the CurseForge maven are not allowed in this project.
Please provide a proper maven repository for this jar.

Reply:
This jar does not have a proper maven repository. I have contacted the devs about this.

Reply:
Then I am sorry, but this can't be merged.
The CurseForge maven is far from reliable and can't be properly proxied via our LDTTeam Buildsytem maven proxies.

Reply:
Why do you need it here at all?

Reply:
I have code in another class that relies on RB to exist within the dev environment to compile properly. Is there a better solution?

Reply:
Depends.

I am not sure how Vanilla MC / Resourcefullbees exposes the items in the hive.

Reply:
RB stores their items in NBT data and provides a method: TieredBeehiveTileEntity.getResourceHoneycomb() which removes one ItemStack from the beehive's storage, and returns it. I also need to import TieredBeehiveTileEntity so that I can check that it's one of RB's hives. Vanilla hives do *not* store items like this, as the only expected drop is honeycombs.

Reply:
Maven is proper now